<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>201</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="digitCount">
         <number>16</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Bin</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_start">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_flash_fw">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Flash</string>
        </property>
       </widget>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
